apps/docs/src/guide/wallets/connectors.md

```diff
@@ -4,6 +4,422 @@
+
+The default behavior will use `LocalStorage` if the `window` is available:
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-options-storage-local{ts:line-numbers}
+
+#### `targetObject`
+
+The `targetObject` is used to provide a target for the `Fuel` manager to interact with. Used for registering events and can be overridden as follows:
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-options-target-object{ts:line-numbers}
+
+### Methods
+
+The `Fuel` manager provides a number of methods to interact with the SDK:
+
+#### All methods from connectors
+
+The `Fuel` manager provides all the [methods](#methods) that are available from the connectors that are currently connected. This means that you can interact with the current connector as if you were interacting with the `Fuel` manager directly.
+
+If there is no current connector available or connected, it will throw an error.
+
+#### `connectors`
+
+The `connectors` method is used to get the current list of _installed_ and _connected_ connectors that are currently available.
+
+<<< @/../../../packages/account/src/connectors/fuel.ts#fuel-sdk-method-connectors{ts:line-numbers}
+
+#### `getConnector`
+
+The `getConnector` method is used to resolve a connector by its name. This is useful for getting a specific connector that you want to interact with. If the connector is not found, it will return `null`.
+
+<<< @/../../../packages/account/src/connectors/fuel.ts#fuel-sdk-method-getConnector{ts:line-numbers}
+
+#### `hasConnector`
+
+The `hasConnector` method will return `true` under the following conditions:
+
+- There is a current connector that is connected.
+- A connector is connected within two seconds of calling the method.
+
+<<< @/../../../packages/account/src/connectors/fuel.ts#fuel-sdk-method-hasConnector{ts:line-numbers}
+
+#### `selectConnector`
+
+The `selectConnector` method accepts a connector name and will return `true` when it is _available_ and _connected_. Otherwise, if not found or is unavailable, it will return `false`.
+
+<<< @/../../../packages/account/src/connectors/fuel.ts#fuel-sdk-method-selectConnector{ts:line-numbers}
+
+#### `currentConnector`
+
+The `currentConnector` method will return the current connector that is connected or if one is available and connected, otherwise it'll return `null` or `undefined`.
+
+<<< @/../../../packages/account/src/connectors/fuel.ts#fuel-sdk-method-currentConnector{ts:line-numbers}
+
+#### `getWallet`
+
+The `getWallet` method accepts an address (string or instance) as the first parameter and a provider or network as the second parameter. It will return an `Account` instance for the given address (providing the address is valid).
+
+If the provider or network is not provided, it will default to the current network. When a provider is unable to be resolved, it will throw an error of code `INVALID_PROVIDER`.
```

Comment:
```suggestion
The provider or network will default to the current network if not provided. When a provider cannot be resolved, it will throw an error of code `INVALID_PROVIDER`.
```

Reply:
09e0871ef593c3cdbf7aa799b437996ef2232e90